docs/3.api/2.composables/third-party-capital/_dir.yml

```diff
@@ -0,0 +1,3 @@
+title: 'Composables'
```

Comment:
![image](https://github.com/nuxt/nuxt/assets/63512348/df4828f4-e8be-4698-803c-51edb180f76d)

We probably can change ti directly to Third Party Capital

Reply:
Do you think it should be moved as the latest ?

(i'm not sure if it would work by prefixing it with a number and a dot  (1. , 2., 3. ...))

Reply:
Do you mean the sub-folder should appear on the bottom as the last entry? 
I do see that the `components` folder uses numbering, which we could try here. 
Not sure how behavior changes between folder & files 🤷

Reply:
the docs is relying on the file order so adding a number should move it to last i think 🤔

Reply:
ok, added a number to the folder... let me know if that works 😄

Reply:
I was thinking to put it as the last one maybe 🤔 @danielroe any thoughts ?

Reply:
do you mean, numbering all files and giving the folder the highest number?

Reply:
yup btw you can see the live preview from the nuxt-studio bot.

hmmm... seems like the icon isn't showing up

![image](https://github.com/nuxt/nuxt/assets/63512348/8e2e85ed-586b-45ac-8eb2-22db58c0873f)

What do you think ? 🤔

Reply:
@huang-julien - I'll undo the folder, according to the comment from @kara and the 'thumbs up' from @danielroe